doc/nrf/ug_nrf5340.rst

```diff
@@ -65,31 +68,36 @@ Available samples
 
 nRF5340 samples consist of two separate images: one that runs on the network core and one that runs on the application core.
 
-Network sample
+Network samples
 ==============
 
-For network communication, Zephyr provides the :ref:`zephyr:bluetooth-hci-rpmsg-sample` sample that implements a Bluetooth Low Energy controller.
-
-This sample must be programmed to the network core to run any Bluetooth Low Energy samples on nRF5340.
+The nRF Connect SDK provides the following samples for the nRF53 network core:
+* :ref:`zephyr:bluetooth-hci-rpmsg-sample` - a Zephyr sample that implements a Bluetooth Low Energy controller.
+  This sample must be programmed to the network core to run standard Bluetooth Low Energy samples on nRF5340.
 
-You might need to adjust the Kconfig configuration of this sample to make it compatible with the peer application.
-For example:
+  You might need to adjust the Kconfig configuration of this sample to make it compatible with the peer application.
+  For example:
 
-* :option:`zephyr:CONFIG_BT_MAX_CONN` must be equal to the maximum number of connections supported by the application sample.
-* If the application sample uses specific Bluetooth LE functionality, this functionality must be enabled in the network sample as well.
+  * :option:`zephyr:CONFIG_BT_MAX_CONN` must be equal to the maximum number of connections supported by the application sample.
+  * If the application sample uses specific Bluetooth LE functionality, this functionality must be enabled in the network sample as well.
   For example, you must modify the configuration of the network sample to make it compatible with the :ref:`ble_throughput` sample::
 
       CONFIG_BT_CTLR_TX_BUFFER_SIZE=251
       CONFIG_BT_CTLR_DATA_LENGTH_MAX=251
 
   This configuration guarantees that the network sample can handle the Bluetooth LE DLE update procedure, which is used in the :ref:`ble_throughput` sample.
 
+* :ref:`peripheral_uart_nrf53` (network part) - a dedicated sample for nRF53 device, showing how the whole BLE stack with a GATT service can run on the network core,
+* :ref:`radio_test` - an application used for testing available modes of the Radio peripheral
+
 Application samples
 ===================
 
 The |NCS| provides a series of :ref:`Bluetooth Low Energy samples <ble_samples>`, in addition to the :ref:`Bluetooth samples in Zephyr <zephyr:bluetooth-samples>`.
 Most of these samples should run on the nRF5340 PDK, but not all have been thoroughly tested.
 Samples that use non-standard features of the Bluetooth Low Energy controller, like the :ref:`ble_llpm` sample, are not supported.
+Additionally, there is a dedicated sample for nRF53 device: :ref:`peripheral_uart_nrf53` (application part) running only application logic while the whole BLE stack including GATT service are running on the network core. Communication with the network MCU is done through a simple, high-level API which is serialized and transported using RPMsg protocol.
```

Comment:
Maybe `RPMsg protocol and TinyCBOR encoder/decoder library`

Reply:
Because _peripheral_uart_nrf53_ was not included in NCS 1.2.0. I removed this sentence.